plutus-tx-plugin/test/TH/Unroll/Golden/comparison.txt

```diff
@@ -0,0 +1,9 @@
+
+┌────────────────────────────┬──────────────────────┬───────────────────┬─────────────┐
+│ Benchmark                  │ CPU units            │ Memory units      │ Size        │
+├────────────────────────────┼──────────────────────┼───────────────────┼─────────────┤
+│ reference                  │ 2182272722  100.00 % │ 8917554  100.00 % │ 68    100 % │
+│ fix-based                  │ 2159024722   98.93 % │ 8772254   98.37 % │ 68    100 % │
+│ manually unrolled 3        │ 2055872722   94.20 % │ 8127554   91.14 % │ 890  1309 % │
```

Comment:
I don't think these numbers are very meaningful by themselves, because they entirely depend on the recursion depth and the unrolling depth. If either of them changes then these numbers will also change.

Reply:
For me the meaning is in seeing that relative numbers (percentages) decrease and a magnitude of the % difference.